### PR TITLE
STRIX-125: add deployment and service for slime chart

### PR DESCRIPTION
**Pull request by cw-sakamoto**
base : https://github.com/cloudposse/charts/tree/master/incubator/monochart

### CODE REVIEW

File: slime/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,96 @@
+{{- if .Values.deployment.enabled -}}
```
Comment (cw-ozaki) [inline slime/templates/deployment.yaml]: https://github.com/chatwork/charts/blob/7a160e4c98b5699e17ab84981e52a8c690f974e3/slime/templates/secret.yaml#L1
こんな感じにドキュメントへの参照が欲しいです。

File: slime/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  strategy:
+{{ toYaml . | indent 4 }}
+{{- end }}
+  revisionHistoryLimit: {{ .Values.deployment.revisionHistoryLimit }}
+  template:
+    metadata:
+      name: {{ include "slime.fullname" $root }}
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        checksum/secret: {{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}
```
Comment (cw-ozaki) [inline slime/templates/deployment.yaml]: I think this should be defined in `values.yaml`.

```
annotation:
{{ .Values.deployment.metadata.annotation | tpl | indent 8 }}
```
If you need to use the helm function, use  `tpl`.

File: slime/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        release: {{ $root.Release.Name | quote }}
+{{- with .Values.deployment.pod.labels }}
+{{ toYaml .| indent 8 }}
+{{- end }}
+    spec:
+{{- with .Values.deployment.pod.affinity }}
+      affinity:
+{{ toYaml . | indent 8 }}
+{{- end }}
+      containers:
```
Comment (cw-ozaki) [inline slime/templates/deployment.yaml]: You should be able to define multiple containers in a slime chart.

```
containers:
{{ tpl .Values.deployment.spec.template.spec.container $root | indent 6 }}
```
Is there any reason for not doing this?

Comment (cw-sakamoto) [reply slime/templates/deployment.yaml]: There is no particular reason, just copying the monochart..🙃
Certainly, this way your suggestion may be more flexible!

File: slime/templates/deployment.yaml (hunk 2 above)
Comment (cw-sakamoto) [reply slime/templates/deployment.yaml]: Same as https://github.com/chatwork/charts/pull/64#discussion_r275184821, just copy monochart...😉

**Comment by cw-ozaki** [comment]
LGTM!

**Pull request merged**